model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -178,11 +202,22 @@ def nodes_builder(model: GraphModule,
                                                  node.args[0].op == PLACEHOLDER and node.args[0].meta[TYPE] in (list, tuple))
             tensor_input_index = []
             op_call_args = list(node.args)
+            for arg_name, arg_val in _args_from_kwargs.items():
+                _args = get_node_args_spec(node)
+                if arg_name in _args:
+                    arg_index = _args.index(arg_name)
+                    # TODO: This code verifies the kwargs with Node values that are used as args are in the correct order.
+                    # (i.e. check that the kwarg with Node is the first of the kwargs)
+                    if len(op_call_args) != arg_index:
+                        Logger.warning("A kwarg with value of Node should be the first after the arguments. "
```

Comment:
what is "value of Node"? (I think I know what it means because you should it to me in the debugger, but it is not clear)